ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-backplane-2.0.yaml

```diff
@@ -0,0 +1,78 @@
+base_images:
+  stolostron_builder_go1.17-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    stolostron_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.17-linux
+  to: cluster-curator-controller
+promotion:
+  name: "2.0"
+  namespace: stolostron
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit-tests
+  commands: make unit-tests
+  container:
+    from: src
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-curator-controller
+    env:
+      IMAGE_REPO: cluster-curator-controller
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: publish
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-curator-controller
+    env:
+      IMAGE_REPO: cluster-curator-controller
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_COMPONENT_NAME=cluster-curator-controller
```

Comment:
please specify the repo & prefix (e.g.
https://github.com/openshift/release/blob/master/ci-operator/config/stolostron/managed-serviceaccount/stolostron-managed-serviceaccount-backplane-2.0.yaml#L86-L87 )